File: client/src/app/models/activity.ts
export interface Activity {
  id: number;
  name: string;
  details: string;
  place: string | null;
  scheduledDateTime: string | null;
  ticketPrice: number | null;
  capacity: number | null;
  available: number | null;
  imageUrl: string | null;
  createdDateTime: string;
}

File: Api/CulturaContigo.Api/Models/Activity.cs
namespace CulturaContigo.Api.Models;

public record Activity
{
    public int Id { get; set; }
    required public string Name { get; set; }
    required public string Details { get; set; }
    public DateTime? ScheduledDateTime { get; set; }
    public string? Place { get; set; }
    public decimal? TicketPrice { get; set; }
    public int? Capacity { get; set; }
    public int? Available { get; set; }
    public string? ImageUrl { get; set; }
}
